### PR TITLE
Descriptive2

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made?
Adds descriptive metadata per @arcadiafalcone's spec.

## Was the documentation (README, wiki) updated?
No.

### CODE REVIEW

File: spec/cocina/models/dro_shared_examples.rb [hunk 1]
```diff
@@ -67,8 +67,7 @@
           description: {
             title: [
               {
-                primary: true,
-                titleFull: 'My object'
+                value: 'My object'
               }
```
Comment (jcoyne) [inline spec/cocina/models/dro_shared_examples.rb]: Should we add `status: “primary”` here?

File: spec/cocina/models/dro_shared_examples.rb [hunk 2]
```diff
@@ -112,11 +111,10 @@
         expect(admin.partOfProject).to eq('Google Books')
 
         desc = instance.description
-        expect(desc.title).to all(be_kind_of(Cocina::Models::Title))
+        expect(desc.title).to all(be_kind_of(Cocina::Models::DescriptiveValueRequired))
         expect(desc.title.size).to eq 1
-        title = desc.title.first
```
Comment (jcoyne) [inline spec/cocina/models/dro_shared_examples.rb]: Should we test for status here?

File: spec/cocina/models/dro_shared_examples.rb (hunk 1 above)
Comment (justinlittman) [reply spec/cocina/models/dro_shared_examples.rb]: Done

File: spec/cocina/models/dro_shared_examples.rb (hunk 2 above)
Comment (justinlittman) [reply spec/cocina/models/dro_shared_examples.rb]: Done

File: lib/cocina/models/validator.rb [hunk 3]
```diff
@@ -4,19 +4,25 @@ module Cocina
...
+
+      # rubocop:disable Style/ClassVars
+      def self.root
+        @@root ||= OpenAPIParser.parse(YAML.load_file(openapi_path))
+      end
+      # rubocop:enable Style/ClassVars
+      private_class_method :root
+
+      def self.openapi_path
+        ::File.expand_path('../../../openapi.yml', ::File.dirname(__FILE__))
```
Comment (jcoyne) [inline lib/cocina/models/validator.rb]: Can this be `__dir__`?

File: lib/cocina/generator/schema.rb [hunk 4]
```diff
@@ -70,7 +65,7 @@ def validate
 
         <<~RUBY
           def self.new(attributes = default_attributes, safe = false, validate = true, &block)
-            Validator.validate(self, attributes.with_indifferent_access) if validate
+            Validator.validate(self, attributes.with_indifferent_access) if validate && self.name
```
Comment (jcoyne) [inline lib/cocina/generator/schema.rb]: is the `self.` necessary?

File: spec/cocina/models/collection_shared_examples.rb [hunk 5]
```diff
@@ -57,8 +57,7 @@
           description: {
             title: [
               {
-                primary: true,
-                titleFull: 'My collection'
+                value: 'My collection'
```
Comment (jcoyne) [inline spec/cocina/models/collection_shared_examples.rb]: Is this still a good example without 'status'?

File: spec/cocina/models/description_spec.rb [hunk 6]
```diff
@@ -3,5 +3,127 @@
...
 
 RSpec.describe Cocina::Models::Description do
-  it 'should have tests', skip: true
+  let(:item) { described_class.new(properties) }
+
+  context 'with minimal required properties provided' do
+    let(:properties) do
+      {
+        title: [{
+          value: 'Tractatus Logico-Philosophicus'
```
Comment (jcoyne) [inline spec/cocina/models/description_spec.rb]: Is this still a good example without `status`?

File: spec/cocina/models/description_spec.rb [hunk 7]
```diff
@@ -3,5 +3,127 @@
...
+    it 'populates required attributes passed in' do
+      expect(item.title.first.value).to eq('Tractatus Logico-Philosophicus')
+    end
+  end
+
+  context 'with optional properties provided' do
+    let(:properties) do
+      {
+        title: [{
+          value: 'Tractatus Logico-Philosophicus'
```
Comment (jcoyne) [inline spec/cocina/models/description_spec.rb]: and here?

File: lib/cocina/generator/schema.rb (hunk 4 above)
Comment (justinlittman) [reply lib/cocina/generator/schema.rb]: No. Removed.

File: lib/cocina/models/validator.rb (hunk 3 above)
Comment (justinlittman) [reply lib/cocina/models/validator.rb]: Yes. Changed.

**Pull request merged**